docs/user-guide/managing-deployments.md

```diff
@@ -411,33 +411,72 @@ You can also run the [update demo](update-demo/) to see a visual representation
 
 ## In-place updates of resources
 
-Sometimes it’s necessary to make narrow, non-disruptive updates to resources you’ve created. For instance, you might want to add an [annotation](annotations.md) with a description of your object. That’s easiest to do with `kubectl patch`:
+Sometimes it’s necessary to make narrow, non-disruptive updates to resources you’ve created. For instance, you might want to update the container's image of your pod.
+
+### kubectl patch
+
+One way to do that is with `kubectl patch`:
 
 ```console
-$ kubectl patch rc my-nginx-v4 -p '{"metadata": {"annotations": {"description": "my frontend running nginx"}}}' 
-my-nginx-v4
-$ kubectl get rc my-nginx-v4 -o yaml
+# use container name "nginx" as a key to update the image from "nginx" to "busybox"
+$ kubectl patch pod my-nginx-1jgkf -p '{"spec":{"containers":[{"name":"nginx","image":"busybox"}]}}'
+"my-nginx-1jgkf" patched
+$ kubectl get pod my-nginx-1jgkf -o yaml
 apiVersion: v1
-kind: ReplicationController
-metadata:
-  annotations:
-    description: my frontend running nginx
+kind: Pod
+...
+spec:
+  containers:
+  -image: busybox
+   name: nginx
 ...
 ```
 
 The patch is specified using json.
 
-For more significant changes, you can `get` the resource, edit it, and then `replace` the resource with the updated version:
+The system ensures that you don’t clobber changes made by other users or components by confirming that the `resourceVersion` doesn’t differ from the version you edited. If you want to update regardless of other changes, remove the `resourceVersion` field when you edit the resource. However, if you do this, don’t use your original configuration file as the source since additional fields most likely were set in the live state.
+
+### kubectl edit
+
+Alternatively, you may also update resources with `kubectl edit`:
+
+```console
+$ kubectl edit pod my-nginx-1jgkf
+```
+
+This is equivalent to first `get` the resource, edit it in text editor, and then `replace` the resource with the updated version:
```

Comment:
Sorry, I didn't mean that we should specify only `EDITOR` here, I was talking in general:) Since we support all three of them, go ahead and add them. Thanks!